source/reporting/index.html.md.erb

```diff
@@ -284,4 +317,142 @@ Example search response:
+* [change what appears on your users' bank statements](#change-what-appears-on-your-users-bank-statements)
+
+#### Add more information to a payment ('custom metadata' or 'reporting columns')
+
+You can use custom metadata to add more information to new payments when creating them through our API. For example, you can add an identifier from your finance or accounting system to help you reconcile the payment later.
+
+Paying users cannot see metadata while they're making a payment.
+
+Include a `metadata` object in the body of your request to [create a new payment](/making_payments/#take-a-payment), for example:
+
+```javascript
+"metadata": {
+   "ledger_code": "AB100",
+   "an_internal_reference_number": 200,
+...
+}
+```
+
+The `metadata` object must contain between 1 and 10 parameters.
+
+Each parameter key must be a unique case-insensitive string between 1 and 30 characters long. If 2 or more keys are identical, the API will remove all but one of the identical keys from the `metadata` object.
+
+The data type of each parameter value must be either a:
+
+- string of no more than 100 characters
+- number
+- boolean
+
+Parameter values can be empty.
+
+The API response will include your metadata.
+
+You cannot add or change metadata keys or values after you've created the payment request.
+
+###### Getting a payment's metadata in the API
+
+When you [get information about a single payment](/reporting/#get-information-about-a-single-payment) or [generate a list of payments](/reporting/#generate-a-list-of-payments-matching-search-criteria), the API response will include the `metadata` object.
+
+The order of the parameters in the `metadata` object may be different to your original object.
+
+##### Get a payment’s settlement information
+
+Payments are ‘settled’ when your PSP sends funds to your bank account. Settlement data helps you reconcile your payments by telling you when an individual payment entered your bank account.
+
+The GOV.UK Pay API can give you payment settlement data in the `settled_date` attribute in `YYYY-MM-DD` format. You can also find payments based on when they were settled.
+
+To get settlement information about a single payment, [get information about that payment](#get-information-about-a-single-payment) through our API.
+
+To get settlement information about multiple payments, [generate a list of payments](#generate-a-list-of-payments-matching-search-criteria) through our API. Use the `from_settled_date` and `to_settled_date` query parameters to only include payments that were settled within a specific date range.
+
+Read more about the `settlement_summary` object in our API reference pages for [getting information about a single payment](/api_reference/single_payment_reference) and [searching for payments](/api_reference/search_payments_reference).
+
+##### Generate a list of payments that match your search criteria
```

Comment:
Is "Generate" the right word here? Why not just "get"?

Reply:
Could this title be something like "Get a list of payments by settled date" or something similar? I thought this was repeating what is higher up on this page, but then I saw this is talking in particular about filtering by settled date.